### ISSUE TITLE
snap: packaging

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/picongpu#1501]
related: #829

forget all we learned, snap seems to be the new kid on the block for shipping ;)

http://snapcraft.io/

**Comment by bussmann** [comment]
So Anaconda's dead?

**Comment by ax3l** [comment]
no, it's orthogonal. snap/docker approaches are light-weight virtualization, conda is packaging in simple environments.
